packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -360,14 +367,23 @@ class Scaffold extends StatefulWidget {
   final List<Widget> persistentFooterButtons;
 
   /// A panel displayed to the side of the [body], often hidden on mobile
-  /// devices.
+  /// devices. Swipes in from left-to-right
```

Comment:
This comment is not accurate in RTL languages.

Reply:
Fixed